src/window_message.cpp

```diff
@@ -126,7 +118,41 @@ void Window_Message::StartMessageProcessing() {
 			}
 		}
 	}
+}
+
+void Window_Message::StartMessageProcessing() {
+	contents->Clear();
+
+	if (Game_Message::is_word_wrapped) {
```

Comment:
Is this variable somehow reset when a new game is started / savegame loaded? Because otherwise the wordwrap logic would be applied to the first displayed msg :D